SonarQube.Client/Api/V8_1/GetHotspotRequest.cs

```diff
@@ -0,0 +1,122 @@
+﻿/*
+ * SonarQube Client
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using SonarQube.Client.Models;
+using SonarQube.Client.Requests;
+
+namespace SonarQube.Client.Api.V8_1
+{
+    public class GetHotspotRequest : RequestBase<SonarQubeHotspot>, IGetHotspotRequest
+    {
+        //TODO - this is an internal API - change to use a public API when available
```

Comment:
I didn't worry too much about the structure of the data that is being returned as I expect it will change in the real API.
For now we just need something that works end-to-end with the currently available APIs, and an interface we can mock in SLVS tests.

Reply:
Are we sure that the API will change? Perhaps we should wait with the merge to master, and instead work on a branch, like with did in the prototype. WDYT?

Reply:
The current API is internal and doesn't return secondary location information, so it seems pretty likely the API will change.
The PR is targeting a branch called `branch-taint`.